smart-cards/src/main/java/gov/va/api/health/smartcards/Compressors.java

```diff
@@ -0,0 +1,53 @@
+package gov.va.api.health.smartcards;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import java.io.ByteArrayOutputStream;
+import java.util.zip.Deflater;
+import java.util.zip.Inflater;
+import lombok.SneakyThrows;
+import lombok.experimental.UtilityClass;
+
+@UtilityClass
+public class Compressors {
+  private static final int BUFFER_SIZE = 1024;
+
+  /** Compress a payload with Java's ZLIB implementation. */
+  public static byte[] deflate(byte[] input) {
+    checkNotNull(input);
+    Deflater deflater = new Deflater();
+    deflater.setInput(input);
+    deflater.finish();
+
+    byte[] buffer = new byte[BUFFER_SIZE];
+    ByteArrayOutputStream bos = new ByteArrayOutputStream();
```

Comment:
even though closing a `ByteArrayOutputStream` has no effect, I feel like this should still be declared via try-with-resources